### DISCUSSION TITLE
How to assign default role for every dashboard in superset

### DISCUSSION THREAD

**Discussion by kalyan4uonly** [apache/superset#22607] [Q&A / Help]
Hi all,

I have enabled DASHBOARD_RBAC, now i am able to see roles menu but none of the role is listed
![image](https://user-images.githubusercontent.com/117740933/210794488-b39b1399-9e7b-4606-b08c-239721d6cddb.png)
For every dashboard i want to assign gamma role by default. How to do this programmatically?
How i will make Gamma as default role for every dashboard?

Can anybody help on this!

Thanks

**Comment by sfirke** [comment]
I don't think this is possible via UI or configuration.

With the caveat that you should back up the database first in case this breaks thing, could you do it by editing the metadata database with a SQL update query that adds the Gamma role ID to every dashboard?
